pynumaflow/datum.py

```diff
@@ -0,0 +1,36 @@
+from datetime import datetime
+
+
+class Datum:
+    """Defines the important information for the event."""
+
```

Comment:
Can we add argument docstring like: https://github.com/numaproj/numalogic/blob/main/numalogic/models/autoencoder/pipeline.py#L19L39

Reply:
Do I still need this kind of docstring after refactoring Datum type back to _dtypes.py ??

Reply:
Added